packages/ra-ui-materialui/src/field/ReferenceField.tsx

```diff
@@ -169,21 +167,12 @@ const stopPropagation = e => e.stopPropagation();
 
 export const ReferenceFieldView: FC<ReferenceFieldViewProps> = props => {
     const {
-        basePath,
         children,
         className,
         error,
```

Comment:
I get Typescript errors:

`TS2339: Property 'error' does not exist on type 'ReferenceFieldViewProps & { children?: ReactNode; }'.`
`TS2339: Property 'isLoading' does not exist on type 'ReferenceFieldViewProps & { children?: ReactNode; }'.`
`TS2339: Property 'referenceRecord' does not exist on type 'ReferenceFieldViewProps & { children?: ReactNode; }'.`

Reply:
Weird. I don't see it and neither does the TypeScript compilation step in the CI. Did you rebuild the ra-core types?

Reply:
You are right. I re-build everything and it works